ulib/FStar.Real.fsti

```diff
@@ -24,8 +24,8 @@ module FStar.Real
   in Z3's theory of real arithmetic.
 *)
 
-val real : eqtype
-
+type real : Type0
+val real_has_eq : squash (hasEq real)
```

Comment:
Why not use just `eqtype` as before?

Reply:
I was working around the bug in thunking ... will fix it now.

Reply:
done